packages/xml-views-tooltip/test/tooltip-spec.ts

```diff
@@ -276,6 +284,15 @@ describe("the UI5 language assistant Hover Tooltip Service", () => {
       expect(ui5Node.kind).equal("UI5Class");
     });
 
+    it("will not get hover content UI5 class with a dot", () => {
```

Comment:
identical test description string appears above.
Mocha uses those strings for uniqueness and it can cause issues

Reply:
It's under a different context